create-token.py

```diff
@@ -0,0 +1,82 @@
+#!/opt/privacyidea/bin/python
```

Comment:
Thats right. The problem with #!/usr/bin/env is, that the user first has to enter the virtual environment so that the script can run. 
So either way is not perfect. I chose /opt/privacyidea since this is the most common scenario.

Reply:
The other script uses the same shebang so I'm ok with that.